src/wren_core.c

```diff
@@ -878,6 +878,24 @@ DEF_PRIMITIVE(num_cos)
   RETURN_NUM(cos(AS_NUM(args[0])));
 }
 
+DEF_PRIMITIVE(num_decimal)
+{
+  double value = AS_NUM(args[0]);
+  RETURN_NUM(value - (int32_t)value);
+}
+
+DEF_PRIMITIVE(num_deg)
+{
+  RETURN_NUM(floor(AS_NUM(args[0]) * 57.2957795130823208768));
+}
+
+DEF_PRIMITIVE(num_div)
+{
+  int32_t left = (int32_t)AS_NUM(args[0]);
+  int32_t right = (int32_t)AS_NUM(args[1]);
+  RETURN_NUM(left / right);
```

Comment:
Like `decimal`, this will also do the wrong thing on > 32-bit values.

That may be OK, if this is what the user wants. I guess it's a question of is this for forced-32-bit integer divide, or just "how many times does the RHS completely divide the LHS?" My intuition is the latter in which case I think you want to just divide the args as doubles and then use `fmod()` to truncate the result.